edgelet/support-bundle/Cargo.toml

```
@@ -10,7 +10,7 @@ chrono = "0.4"
 futures = "0.3"
 hyper = { version = "0.14", features = ["stream"] }
 thiserror = "1"
-tokio = { version = "1", features = ["process", "time"] }
+tokio = { version = "1", features = ["parking_lot", "process", "time"] }
```

```suggestion
tokio = { version = "1", features = ["process", "time"] }
```
This crate does not use `tokio::sync::..`.